bedrock/base/templates/includes/affiliate-banner.html

```diff
@@ -0,0 +1,25 @@
+{#
+  This Source Code Form is subject to the terms of the Mozilla Public
+  License, v. 2.0. If a copy of the MPL was not distributed with this
+  file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+<aside class="mzp-c-notification-bar affiliate-notification hide-from-legacy-ie">
+  {% if not affiliate_message %}
+    {% set attrs = 'href="' ~ url('privacy.notices.websites') ~ '" target="_blank" rel="noopener noreferrer"' %}
```

Comment:
This is left over from the existing banner, but we can remove `rel="noopener noreferrer"` for same-site links?